packages/web-components/src/components/cbp-toast/cbp-toast.scss

```diff
@@ -56,11 +56,10 @@
 
 cbp-toast {
   display: flex;
-  margin-left: auto;
-  margin-bottom: var(--cbp-space-6x);
-  width: max-content;
-  max-width: 50%;
-  background-color: var(--cbp-toast-background);
+  margin-inline-start: auto;
+  margin-block-end: var(--cbp-space-6x);
+  width: min(100%, 500px);
```

Comment:
I'd still use relative units instead of px. Convert to rems in a round number like 30rem *(480px) or similar.

Reply:
> updated to 30rem